### PR TITLE
New trans

### PR DESCRIPTION
**Pull request by reginafcompton**

### CODE REVIEW

**Comment by reginafcompton** [comment]
@jeancochrane - lots of new tags added. Could you look it over?

**Comment by jeancochrane** [comment]
Ready for review @reginafcompton?

**Comment by reginafcompton** [comment]
Yes! Finally defeated Travis.

File: templates/base.html [hunk 1]
```diff
@@ -100,8 +100,14 @@
     // Attach click event to dropdown, which reloads page with new language code 
     transObj = {
       'French': 'fr',
```
Comment (jeancochrane) [inline templates/base.html]: What stops us from doing, say:

```javascript
transObj = {
    '{% trans "French" %}': 'fr',
    '{% trans "English" %}': 'en',
    '{% trans "Spanish" %}': 'es'
}
```

File: templates/sfm/dashboard.html [hunk 2]
```diff
@@ -5,32 +5,33 @@
     <div class="col-sm-12">
         <div class="jumbotron">
             <form action='{% url 'search' %}' method='GET'>
-                <p>Investigate <strong>police</strong> and <strong>military commanders</strong>, <strong>units</strong> and <strong>areas of operation</strong></p>
+                {% comment %}Translators: Please keep the "strong" tags, which display the numbers in bold{% endcomment %}
```
Comment (jeancochrane) [inline templates/sfm/dashboard.html]: This is cool! Were you able to confirm that it works?

File: templates/violation/detail.html [hunk 3]
```diff
@@ -12,36 +12,36 @@
 
 {% block header %}
     <h2 class="page-title">
-        <i class="fa fa-fw fa-exclamation-triangle"></i> Incident
+        <i class="fa fa-fw fa-exclamation-triangle"></i> {% trans "Incident" %}
         {% if violation.geoname %}
-            in <strong>{{ violation.geoname }}</strong>
+            {% trans "in" %} <strong>{{ violation.geoname }}</strong>
```
Comment (jeancochrane) [inline templates/violation/detail.html]: We might want comments for these isolated prepositions letting the translator know what the different kinds of output of the string are supposed to look like:

`Incident on <date>`
`Incident between <date> and <date>`
`Incident starting <date>`
`Incident ending <date>`

File: templates/base.html (hunk 1 above)
Comment (reginafcompton) [reply templates/base.html]: Oh, geeze...right..

File: templates/sfm/dashboard.html (hunk 2 above)
Comment (reginafcompton) [reply templates/sfm/dashboard.html]: Yes, it does!

File: templates/violation/detail.html (hunk 3 above)
Comment (reginafcompton) [reply templates/violation/detail.html]: Nice thought!

**Pull request merged**